Examples don't load after running `npm install` and `npm start`

After cloning the repository and then running `npm install` and `npm start` and navigating to `http://localhost:3000` the list of examples does load, but after clicking on any of them, an error like this is shown in the browser console and the loading text remains on the screen indefinitely
```
client?843a:48 multi ./demo/src/tile
Module not found: Error: Cannot resolve 'file' or 'directory' ./demo/src/./demo/src/tile/app.component.ts in C:\Users\brad.woods\Documents\sky-pages\skyux2
resolve file
  C:\Users\brad.woods\Documents\sky-pages\skyux2\demo\src\demo\src\tile\app.component.ts doesn't exist
  C:\Users\brad.woods\Documents\sky-pages\skyux2\demo\src\demo\src\tile\app.component.ts.ts doesn't exist
  C:\Users\brad.woods\Documents\sky-pages\skyux2\demo\src\demo\src\tile\app.component.ts.js doesn't exist
resolve directory
  C:\Users\brad.woods\Documents\sky-pages\skyux2\demo\src\demo\src\tile\app.component.ts doesn't exist (directory default file)
  C:\Users\brad.woods\Documents\sky-pages\skyux2\demo\src\demo\src\tile\app.component.ts\package.json doesn't exist (directory description file)
[C:\Users\brad.woods\Documents\sky-pages\skyux2\demo\src\demo\src\tile\app.component.ts]
[C:\Users\brad.woods\Documents\sky-pages\skyux2\demo\src\demo\src\tile\app.component.ts.ts]
[C:\Users\brad.woods\Documents\sky-pages\skyux2\demo\src\demo\src\tile\app.component.ts.js]
 @ multi ./demo/src/tile
```

--- a/config/webpack/webpack.demo-visual.js
+++ b/config/webpack/webpack.demo-visual.js
@@ -14,7 +14,7 @@ var entry = {};
     var parts;
 
     dirname = path.dirname(file);
-    parts = dirname.split(path.sep);
+    parts = dirname.split('/'); // glob always returns '/' as separator
     demoName = parts[parts.length - 2];
     entry[demoName] = file;
 

--- a/config/webpack/webpack.demo.js
+++ b/config/webpack/webpack.demo.js
@@ -14,7 +14,7 @@ var entry = {};
     var parts;
 
     dirname = path.dirname(file);
-    parts = dirname.split(path.sep);
+    parts = dirname.split('/'); // glob always returns '/' as separator
     demoName = parts[parts.length - 1];
 
     entry[demoName] = './demo/src/' + demoName + '/app.component.ts';

